File: Data Structures & Algorithms/pow-x-n/submission-0.py
class Solution:
    def myPow(self, x: float, n: int) -> float:
        pr=1.00000
        if n>0:
            while n:
                pr*=x
                n=n-1
        if n<0:
            n=-1*n
            while n:
                pr/=x
                n=n-1
        return pr
            
        

File: Data Structures & Algorithms/set-zeroes-in-matrix/submission-0.cpp
class Solution {
public:
    void setZeroes(vector<vector<int>>& matrix) {

        int n = matrix.size();
        int m = matrix[0].size();

        vector<int> rows;
        vector<int> cols;

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {

                if (matrix[i][j] == 0) {
                    rows.push_back(i);
                    cols.push_back(j);
                }
            }
        }
        for (int r : rows) {
            for (int j = 0; j < m; j++) {
                matrix[r][j] = 0;
            }
        }

        for (int c : cols) {
            for (int i = 0; i < n; i++) {
                matrix[i][c] = 0;
            }
        }
    }
};